frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/title/title_node_widget.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:appflowy/workspace/application/view/view_service.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../../../application/doc_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+const String kTitleType = 'title';
+
+class TitleNodeWidgetBuilder extends NodeWidgetBuilder {
+  TitleNodeWidgetBuilder({required this.docBloc});
```

Comment:
If all fields are final, can't this constructor be const?

Reply:
I get a warning about removing the const keyword. Would it be better to just not make the field not final?

Reply:
Actually, it's because `NodeWidgetBuilder` does not have a const constructor.